.gitignore

```diff
@@ -63,3 +63,5 @@ xcuserdata/
 # Emacs files
 TAGS
 .dir-locals.el
+build_lux_osx.sh
+pack_lux_osx_2.sh
```

Comment:
Shouldn't be here a "scripts/" in front of both new lines ?

Reply:
No i moved them and now they are just local